README.md

```diff
@@ -26,14 +26,26 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] Mention two parts of Express that you learned about this week.
 
+routing which allows us to build several endpoints to our server based on the url
+process.env which allows us to provide environment variables
```

Comment:
environment variables are extremely useful as you will when you learn more about them this week